common/lib/xmodule/xmodule/conditional_module.py

```diff
@@ -95,27 +143,15 @@ class ConditionalModule(ConditionalFields, XModule):
         'voted': 'voted'  # poll_question attr
     }
 
-    def _get_condition(self):
```

Comment:
Thinking this through some more, the logic would have to ignore the specified ``conditional_value`` field in this case too, as that won't match for pre-existing courses.